ansible/configs/satellite-multi-region/sample_vars.yml

```diff
@@ -88,6 +91,27 @@ satellite_repository:
     name: 'Red Hat Satellite Tools 6.4 (for RHEL 7 Server) (RPMs)'  
```

Comment:
Shouldn't the '6.4' be parameterised?

Reply:
This is sample_vars.yml configured to install 6.4 only if some one wants then it can be changed